package.json

```diff
@@ -90,6 +90,7 @@
     "js-yaml": "^3.14.0",
     "json-stringify-pretty-compact": "^2.0.0",
     "jsonschema": "^1.1.0",
+    "lodash": "^4.17.15",
```

Comment:
This was already used as a dependency, so I added it as a dev dependency and used it, as `lodash-es` doesn't work because rollup doesn't transpile `node_modules`.